Tinyxml2Serializer.cpp

```diff
@@ -37,33 +37,42 @@ void CTinyxml2Serializer::AddXMLElementInt(tinyxml2::XMLDocument* oXML, const ch
     //}
 }
 
-void CTinyxml2Serializer::AddXMLElementUnsignedInt(tinyxml2::XMLDocument* oXML, const char* tTag, const unsigned int value)
+void CTinyxml2Serializer::AddXMLElementUnsignedInt(tinyxml2::XMLElement* parent, const char* tTag, const unsigned int nValue, tinyxml2::XMLDocument* oXML)
```

Comment:
Suggestion: Change pointer types of the xml objects to references and null-checks.

Reply:
Fixed!